### ISSUE TITLE
Missing attribute aria-label on Edit Dialogs

### ISSUE THREAD

**Issue by ferchoman09** [daniel-nagy/md-data-table#525]
I have a google chrome warning when I use 
$mdEditDialog.small(options);

ARIA: Attribute " aria-label ", required for accessibility, is missing on node: angular.min.js:117 

It's a minor bug but is good an aplication without warnings.

**Comment by tstrohmeier** [comment]
The same problem with $mdEditDialog.large(options); here.

**Comment by offwork** [comment]
I think the same problem continues... [http://prnt.sc/e3vogg](url)  It would be enough to add "aria-label" for the inputs! We are waiting....

**Comment by ielektronick** [comment]
As a temporary fix, you could use `validators` property:
`...validators: { 'md-maxlength': 50, 'aria-label': 'Your aria label' } ...`
